ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -103,6 +105,20 @@ class TabBarViewDemoController: DemoController {
         updateBadgeNumbers()
     }
 
+    private func homeItem(shouldShowTitle: Bool) -> TabBarItem {
```

Comment:
You can use this for the initialization of the property too. That way you're avoiding duplicated code.